portfolio/src/main/webapp/about.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>About Me</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+    <style>
+    a    {
+        color: black;
+        text-shadow: 2px 2px 5px turquoise, 1px 1px 3px purple, 1px 2px 3px snow;
+        text-decoration: underline;
+    }
+    body {
+        background: url(images/better.jpg) no-repeat center center fixed;
+    }
+
+    html {
+        scroll-behavior: smooth;
+    }
+    </style>
+  </head>
+  <body>
+    <div id="content">
+        <div style="border-bottom:3px solid #000">
+            <a href="index.html">
+                <img src="/images/home.webp" width="50" height="50" alt="Home">
```

Comment:
Similar to above comment, the width and height should be preferred to be in a stylesheet that has a rule targeting this image since it is a visual aspect. This instance is a little more arguable than using the style attribute though (most developers will probably say "ehh.... its fine", Im more of a stickler for code style)